consensus/clique/clique.go

```diff
@@ -378,7 +378,7 @@ func (c *Clique) verifyCascadingFields(chain consensus.ChainReader, header *type
 		parent = chain.GetHeader(header.ParentHash, number-1)
 	}
 
-	// Added to bypass the situation when the parent is missing in celolatest mode
+	// Added to bypass the situation when the parent is missing in ultralight mode
```

Comment:
To that point, we would have previously been entering this when in ultralight mode, right?

Reply:
Yes, the comment was incomplete earlier.